generate_trade_report.py

```
@@ -12,7 +12,7 @@
 
 from openpyxl import load_workbook
 
-SRC=Path('ReportHistory-2002705608.xlsx'); OUT=Path('reports'); OUT.mkdir(exist_ok=True)
+SRC=Path('source/ReportHistory-2002705608.xlsx'); OUT=Path('reports'); OUT.mkdir(exist_ok=True)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)